remail/email_api/service.py

```
@@ -0,0 +1,510 @@
+    
+    def get_emails(self, date : datetime = None)->list[Email]:
+        if not self.logged_in: 
+            raise ee.NotLoggedIn()
+        listofMails = []
+        self.IMAP.select_folder("INBOX")
+        if date is not None:
+            messages_ids = self.IMAP.search([u'SINCE', date])
+        else: 
+            messages_ids = self.IMAP.search(["ALL"])
+
+        for msgid,message_data in self.IMAP.fetch(messages_ids,["RFC822"]).items():
```

Ruff (0.8.2)

Loop control variable `msgid` not used within loop body

Rename unused `msgid` to `_msgid`

(B007)